cordova-plugin-outline/apple/src/OutlinePlugin.swift

```diff
@@ -295,4 +299,59 @@ class OutlinePlugin: CDVPlugin {
     }
     return callbackId
   }
+
+  // Migrates local storage files from UIWebView to WKWebView.
+  private func migrateLocalStorage() {
+    let appLibraryDir = FileManager.default.urls(for: .libraryDirectory, in: .userDomainMask)[0]
+
+    var uiWebViewLocalStorage: URL
+    if FileManager.default.fileExists(atPath: appLibraryDir.appendingPathComponent("WebKit/LocalStorage/file__0.localstorage").relativePath) {
+        uiWebViewLocalStorage = appLibraryDir.appendingPathComponent("WebKit/LocalStorage")
+    } else {
+        uiWebViewLocalStorage = appLibraryDir.appendingPathComponent("Caches")
+    }
+
+    // Local storage backing files have the following naming format: $scheme_$hostname_$port.localstorage
+    // With UIWebView, the app used the file:// scheme with no hostname and any port
+    uiWebViewLocalStorage = uiWebViewLocalStorage.appendingPathComponent("file__0.localstorage")
```

Comment:
Let's not modify the variable. It makes it harder to understand. Perhaps call the original one `uiWebViewLocalStorageDir`?

Reply:
Done in bc645f8.